### PR TITLE
Feat: algolia search multi version in documentation

### PR DESCRIPTION
**Pull request by toofff**
Updating the search configuration for the [API platform website](https://api-platform.com/).
  - The default search is only on the current version.
  - We would like to search by facet according to a version.

Pull request [Algolia configuration](https://github.com/algolia/docsearch-configs/pull/876)

### CODE REVIEW

File: src/components/layout/Search.js [hunk 1]
```diff
@@ -27,6 +27,9 @@ class Search extends React.Component {
         indexName: process.env.GATSBY_DOCSEARCH_INDEX_NAME,
         inputSelector: this.searchInput,
         debug: false,
+        algoliaOptions: {
+          'facetFilters': ["version:2.4"]
```
Comment (alanpoulain) [inline src/components/layout/Search.js]: You should use `currentVersion` from `constants.js`.

**Comment by alanpoulain** [comment]
Can we merge it now or should we wait for the other PR to be merged?

**Comment by toofff** [comment]
@alanpoulain It is necessary to wait until my tests is conclusive on the other pull request

**Comment by toofff** [comment]
@dunglas @alanpoulain 

With this Pull Request the search is based only on the current version.
I still have to look for the search by version.

But for the moment this is mergeable ;)

**Comment by dunglas** [comment]
Thanks @toofff! LGTM

**Pull request merged**